.claude/skills/radio-link-testing/scripts/gen_patch_seq.py

```
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+"""
+gen_patch_seq.py – Saratoga-style hole detection for a received file segment.
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)